.github/actions/setup-backend/action.yml

```diff
@@ -26,11 +26,12 @@ runs:
       shell: bash
       run: |
         if [ "${{ inputs.python-version }}" = "current" ]; then
-          echo "PYTHON_VERSION=3.10" >> $GITHUB_ENV
+          echo "PYTHON_VERSION=3.11.11" >> $GITHUB_ENV
         elif [ "${{ inputs.python-version }}" = "next" ]; then
+          # TODO: set to 3.12 when libs are ready - seemed hard on 12/17/24
           echo "PYTHON_VERSION=3.11" >> $GITHUB_ENV
```

Comment:
Aren't these reversed, i.e. `current` should point to `3.11` and `next` should point to `3.11.11`?